apps/ledger-live-mobile/src/components/GenericErrorView.tsx

```diff
@@ -28,8 +28,8 @@ type Props = {
 };
 
 const StyledLink = styled(Link)`
-  margin-top: ${p => p.theme.space[8]};
-  margin-bottom: ${p => p.theme.space[4]};
+  margin-top: ${p => p.theme.space[8]}px;
+  margin-bottom: ${p => p.theme.space[4]}px;
```

Comment:
[ASK] I don't really know how it works in react native but for android we are using dp (density pixel) as we have several (> 6000) existing screens configuration (size, resolution, quality). It's really not recommanded to use pixels.

Reply:
In react native styled components, `px` gets interpreted as `dp` 👍 
Putting no unit results in a warning (which is what I'm fixing there).
`dp` or `dip` are not valid units for styled components:
<img width="695" alt="Screenshot 2023-12-11 at 17 50 16" src="https://github.com/LedgerHQ/ledger-live/assets/91890529/979589b9-5fc7-4f98-a9a7-5e6bdc58279f">

So px is the only "valid" unit to put there, even if it's not actually pixels but density-independent pixels 🤷‍♂️